totalRP3_Extended/dialogues/dialogues.lua

```diff
@@ -243,11 +245,13 @@ local function playDialogStep()
 				-- End point
 				dialogFrame.Chat.NextButton:SetScript("OnClick", finishDialog);
 			else
+				print(dialogFrame.stepIndex)
```

Comment:
bruh

Reply:
see, i said it needed additional tests :)